### DISCUSSION TITLE
[Feature] Is it possible to provide UI to config alarm rule, or already have one, but i don't know

### DISCUSSION THREAD

**Discussion by johnxue2013** [apache/skywalking#9164] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

HI，as i have read Alarm chapter (https://skywalking.apache.org/docs/main/v8.5.0/en/setup/backend/backend-alarm/), but i don't find how to config a alarm rule by UI , only way is to edit the alarm config file, if any one kowns, that will be nice

### Use case

if there a method to edit alarm rule by UI, rather then logon on the skywakling oap server to edit alram file directly

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment, marked as answer]
No, there isn't. There are various ways to set configurations, SkyWalking doesn't provide management features according to community consensus.
